activerecord/test/cases/fixtures_test.rb

```diff
@@ -366,11 +366,18 @@ class FixturesResetPkSequenceTest < ActiveRecord::TestCase
     fixtures :accounts
     fixtures :companies
 
-    def setup
```

Comment:
If we are using `teardown` with block I'd change to `setup` with block. I personally prefer the method calling `super`.